src/hooks/useOperatorTrades.ts

```diff
@@ -1,68 +1,44 @@
-import { Trade as TradeMetaData } from '@cowprotocol/cow-sdk'
-import { getTrades, Order, Trade } from 'api/operator'
+import { getTrades, Order, RawTrade, Trade } from 'api/operator'
 import { useCallback, useEffect, useState } from 'react'
 import { useNetworkId } from 'state/network'
 import { Network, UiError } from 'types'
 import { transformTrade } from 'utils'
-
-type Params = {
-  owner?: string
-  orderId?: string
-}
+import { web3 } from 'apps/explorer/api'
 
 type Result = {
   trades: Trade[]
   error?: UiError
   isLoading: boolean
 }
 
-/**
- * Fetches trades for given filters
- * When no filter is given, fetches all trades for current network
- */
-export function useTrades(params: Params): Result {
-  const { owner, orderId } = params
+type TradesTimestamps = { [txHash: string]: number }
 
-  const [isLoading, setIsLoading] = useState(false)
-  const [error, setError] = useState<UiError>()
-  const [trades, setTrades] = useState<Trade[]>([])
+const tradesTimestampsCache: { [blockNumber: number]: number } = {}
 
-  // Here we assume that we are already in the right network
-  // contrary to useOrder hook, where it searches all networks for a given orderId
-  const networkId = useNetworkId()
+async function fetchTradesTimestamps(rawTrades: RawTrade[]): Promise<TradesTimestamps> {
+  const requests = rawTrades.map(({ txHash, blockNumber }) => {
+    const cachedValue = tradesTimestampsCache[blockNumber]
 
-  const fetchTrades = useCallback(async (networkId: Network, owner?: string, orderId?: string): Promise<void> => {
-    setIsLoading(true)
+    if (cachedValue) {
+      return { txHash, timestamp: cachedValue }
+    }
 
-    try {
-      let trades: TradeMetaData[] = []
-      if (orderId) {
-        trades = await getTrades({ networkId, orderId })
-      } else if (owner) {
-        trades = await getTrades({ networkId, owner })
-      }
+    return web3.eth.getBlock(blockNumber).then(res => {
```

Comment:
I think we have different linter rules. Not sure if mine or yours is the good one 
<img width="522" alt="image" src="https://user-images.githubusercontent.com/2352112/230348747-4b708e6f-e3fe-425f-b608-cf17e24b24a4.png">

Reply:
Thanks! Fixed